### ISSUE TITLE
Batch Normalization slow?

### ISSUE THREAD

**Issue by dansileshi** [tensorflow/tensorflow#4208]
I am using tf.contrib.layers.batch_norm
This is very slow. Any better implementation?

**Comment by tatatodd** [comment]
We primarily use github issues for bugs and installation issues.  This is a question better suited for StackOverflow.  Please ask it there and tag it with the `tensorflow` tag.  Thanks!

FYI you might be interested in https://github.com/tensorflow/models/issues/325, which says that 0.10.0rc0 has some known performance regressions which should be fixed by the actual release.
